src/test/wdio.conf.js

```diff
@@ -29,6 +29,20 @@ exports.config = {
    * webdriver service
    */
   services: ['selenium-standalone'],
+  seleniumArgs: {
+    drivers : {
+      chrome : {
+        version : "77.0.3865.40"
+      }
+    }
+  },
+  seleniumInstallArgs: {
+    drivers : {
+      chrome : {
+        version : "77.0.3865.40"
```

Comment:
this should be variable. travis install latest version

Reply:
the code was removed from this PR